lib/chef/provider/aws_auto_scaling_group.rb

```diff
@@ -6,7 +6,7 @@ class Chef::Provider::AwsAutoScalingGroup < Chef::Provisioning::AWSDriver::AWSPr
   protected
 
   def create_aws_object
-    converge_by "create new Auto Scaling Group #{new_resource.name} in #{region}" do
+    converge_by "create Auto Scaling group #{new_resource.name} in #{region}" do
```

Comment:
Even the console is divided on this - sometimes I see `Group` and sometimes I see `group`.  But it looks like the `G` is only in titles and navigation bars, which typically use title casing anyways.  I'm declare it shall be `g`roup!